pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -61,23 +62,23 @@ For your entry point to complete these tasks, you will need to add the following
 ```scss
 /* styles.scss */
 
-// 1. Load USWDS settings
-@forward "uswds-settings.scss";
+// 1. Load your project's USWDS settings configuration
+@forward "uswds-theme.scss";
 
 // 2. Load USWDS source code
 @forward "./path/to/source/uswds";
 
 // 3. Load your project's custom Sass
-@forward "project-custom.scss";
+@forward "uswds-theme-custom-styles.scss";
```

Comment:
The previous `project-custom.scss` seems clearer that they are project specific styles.

Reply:
That makes sense. I updated the name to `project-styles.scss` in [4661914](https://github.com/uswds/uswds-site/pull/2123/commits/4661914f8f766e16e80bc1b5f1f648aad2d1783b) to be more descriptive. Does that work for you?